Integrations/QRadar/QRadar.py

```diff
@@ -525,8 +525,9 @@ def create_incident_from_offense(offense):
     labels = []
     for i in range(len(keys)):
         labels.append({'type': keys[i], 'value': convert_to_str(offense[keys[i]])})
+    formatted_description = offense['description'].replace('\n', '') if offense['description'] else ''
```

Comment:
```suggestion
    formatted_description = offense['description'].replace('\s\n', ' ').replace('\n', ' ') if offense['description'] else ''
```

this way you will replace the new lines with space and will concat words(or create double spaces)
what do you think?

Reply:
Good idea, this should work but with `re.sub(r'\s\n', ' ', s)`. I'll commit that change.